pkg/stack/unwind/unwind_table.go

```diff
@@ -284,14 +300,14 @@ type UnwindTableRow struct {
 	RA frame.DWRule
 }
 
-func buildTableRows(fde *frame.FrameDescriptionEntry) []UnwindTableRow {
-	rows := make([]UnwindTableRow, 0)
-
-	frameContext := frame.ExecuteDwarfProgram(fde)
-
-	instructionContexts := frameContext.InstructionContexts()
+func nextUnwindRow(fde *frame.FrameDescriptionEntry) func() *UnwindTableRow {
```

Comment:
@javierhonduco this one ☝️

Reply:
I'm not sure how to rewrite this without passed or implicit callbacks. Need to put more thought into this. Will take a look next week when I'm back. If I can't figure it out will let you know. I might be missing some insights

Reply:
Wasn't very inspired last week I guess :D Added some comment in the PR